HealthTracker/feature/BloodPressureForm/validationschema.ts

```diff
@@ -0,0 +1,22 @@
+import * as yup from "yup";
+
+export const validationSchema = yup.object().shape({
+  sys: yup
+    .number()
+    .typeError("The value must be a number")
```

Comment:
Move the magic strings into a constant file, this will help to implement i18n in the feature

Reply:
done